driverinference/main.py

```diff
@@ -10,7 +10,7 @@
 
 
 class Inference:
- 
+
```

Comment:
maybe add language to run arguments in output like below

Run Arguments:
        constraint = P100
        nodes = 1
        ntasks = 18
        ntasks-per-node = 18
        language = ur mom

Reply:
I see it in the Executable path tho, but would be nice to have it right in my face